app/policies/booking_policy.rb

```diff
@@ -0,0 +1,35 @@
+class BookingPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+  end
+
+  def index?
+    record.user == user
+  end
+
+  def show?
+    record.user == user
+  end
+
+  def create?
+    true
+  end
+
+  def new?
+    create?
+  end
+
+  def update?
```

Comment:
I don't believe you need any of these methods: update? edit? destroy?
since these routes and controller actions won't exist